scripts/update-yarn-locks.sh

```diff
@@ -0,0 +1,20 @@
+#! /bin/bash
+set -ueo pipefail
+
+# This script updates all yarn.lock files in the repository, except for the one
+# in the root directory.
+find . -name node_modules -prune -o -name yarn.lock -print0 \
+  | while IFS= read -r -d $'\0' lockfile; do
+    dir="$(dirname "$lockfile")"
+    test "$dir" != . || continue
+    echo "Updating $lockfile"
+    (
+      cd "$dir"
+      if test "${COREPACK_ROOT+set}" = set; then
+        "$COREPACK_ROOT/shims/yarn" install
+      else
+        yarn install
+      fi
```

Comment:
I'm curious, why is this necessary. If corepack is enabled, wouldn't it be in the `PATH` ?

Reply:
If corepack is enabled, the `yarn` it has in its PATH is currently the `"packageManager": "yarn@1.22.22"` in the `agoric-sdk/package.json`.  Since we're executing in a directory with a different `"packageManager": "yarn@4..."`, we need to reevaluate the corepack-shimmed yarn to dispatch to the `yarn@4` `packageManager`.

That, of course, demands a comment.